packages/web/components/LoginButton.tsx

```diff
@@ -45,16 +46,25 @@ export const LoginButton: React.FC = () => {
             {user.player ? getPlayerName(user.player) : 'Unknown'}
           </MetaLink>
           <HStack spacing={2}>
-            <MetaLink href="/profile/setup">Setup profile</MetaLink>
+            <MetaLink
```

Comment:
Why these changes in this branch ?

Reply:
When you narrow the screen when the profile options are written out they are wide enough they push the profile picture behind the drawer. The switch to icons was to narrow everything.